Add More Loadout Options

Haltell opened the pull request:
Added new loadout options for Captain, Chemist, Passenger, and Security.

Added Filled Jack Boots for security loadouts.

<!-- Please read these guidelines before opening your PR: https://docs.spacestation14.io/en/getting-started/pr-guideline -->
<!-- The text between the arrows are comments - they will not be visible on your PR. -->

## About the PR
<!-- What did you change in this PR? -->
Added some extra loadout options from DeltaV/Nyanotrasen files.
Open to any other reasonable options for loadouts.

Current List of Additions:
Scarfs added to most roles, with the following exceptions:
- Assistant Roles
- Captain
- Chef
- Mantis
- Prison Guard
- Station Specific Jobs (except Corpsman)

Command:
- Captain & HoP's Inspection Gloves
- Captain & HoP's Brown, White, Lace Up, and Winter Shoes
- HoP's Outer Clothes
- HoP's Uniforms
- Captain's Beret
- All of Command has their winter boots as options

Medical:
- Paramedic Winter Boots given to Paramedic
- Stethoscopes given to CMO, Doctors, and Paramedics
Chemist:
- Heavy Nitrile Gloves
- Chemical Resistant Apron
- Enclosed Shoes
- Replaced Medical Winter Boots with Chemist Variant

Bartender:
- Changed glasses options. Added beer goggles.

Passenger:
- Blue, Red, Purple Casual Jumpsuit/skirt
- Yellow Skirt with Rose Hoodie
- Turquoise Hoodie and Shorts

Engineering:
- Removed requirement for hats
- Hats given to Atmospherics

Epistemics:
- Roboticist Labcoat for Scientists

Security:
- Jackboots
- Durathread Vest for Officers
- Red Beret for Officers and Cadets
- Blue Jumpsuit/skirt for Officers

Logistics:
- LO's DeltaV Beret
- Salvage Cloak + Miner Boots given to Salvage


## Why / Balance
<!-- Why was it changed? Link any discussions or issues here. Please discuss how this would affect game balance. -->
Extra choices for characters to spend more time RPing and less time walking to, and standing in front of a clothes vending machine.

## Technical details
<!-- If this is a code change, summarize at high level how your new code works. This makes it easier to review. -->

## Media
<!-- 
PRs which make ingame changes (adding clothing, items, new features, etc) are required to have media attached that showcase the changes.
Small fixes/refactors are exempt.
Any media may be used in SS14 progress reports, with clear credit given.

If you're unsure whether your PR will require media, ask a maintainer.

Check the box below to confirm that you have in fact seen this (put an X in the brackets, like [X]):
-->
![image](https://github.com/DeltaV-Station/Delta-v/assets/15059141/50f34e49-a8d9-42ae-adc0-6634058ce662)
![image](https://github.com/DeltaV-Station/Delta-v/assets/15059141/33af3dba-6525-453f-bd5c-426f6d6acbde)
![image](https://github.com/DeltaV-Station/Delta-v/assets/15059141/bf0c4612-a8b1-43b5-9f30-b144673a1100)
![image](https://github.com/DeltaV-Station/Delta-v/assets/15059141/5cd4e40a-2a40-4b66-bc83-2ac99afc58db)


- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes
<!--
List any breaking changes, including namespace, public class/method/field changes, prototype renames; and provide instructions for fixing them. This will be pasted in #codebase-changes.
-->

**Changelog**
<!--
Make players aware of new features and changes that could affect how they play the game by adding a Changelog entry. Please read the Changelog guidelines located at: https://docs.spacestation14.io/en/getting-started/pr-guideline#changelog
-->

<!--
Make sure to take this Changelog template out of the comment block in order for it to show up.
:cl:
--!>
- add: Nanotrasen uniform stores resotcked; many more options for your style!

Haltell committed:
Add More Loadout Options

Added new loadout options for Captain, Chemist, Passenger, and Security.

Added Filled Jack Boots for security loadouts.

DangerRevolution commented on Resources/Prototypes/Loadouts/role_loadouts.yml:
```diff
@@ -346,10 +346,10 @@
   groups:
   - MedicalMask
   - ChemistJumpsuit
-  - MedicalGloves
```
This needs to be commented out, and replaced with Chemist gloves *or* this needs to have a comment saying "Delta-V : Replaces MedicalGloves/Shoes with CHemistGloves/Shoes"

Haltell replied:
Thanks, just fixed up all three of these things :)

DangerRevolution commented on Resources/Prototypes/Loadouts/role_loadouts.yml:
```diff
@@ -346,10 +346,10 @@
   - ChemistBackpack
   - ChemistOuterClothing
-  - MedicalShoes
+  - ChemistShoes # DeltaV
```
This needs to be commented out, and replaced with Chemist gloves *or* this needs to have a comment saying "Delta-V : Replaces MedicalGloves/Shoes with CHemistGloves/Shoes"

DangerRevolution commented:
Changelog should also mention that Chemists now start with Chemist equivalents of shoes & gloves instead of Medical versions

rosieposieeee commented:
red beret for all of sec or riot

Adrian16199 commented:
Give a red scarf to security, blue scarf to medical, green scarf to botany, yellow scarf to engineering, brown to logistics. No matter the role, just to all of them to said department.
Then this will be a good PR.

Timemaster99 commented:
I made a list in the discord of other things requiring attention in loadouts. Since you are open to further additions, I'll paste it here.

- HoP is missing their turtleneck, mess uniform, and their choice between durathread vest, armored coat, or jacket
- LO has the Wizden logistic's officer beret instead of ours
- MG's cloak sprite got overwritten by the Wizden one 
- Sec jobs have the two deprecated armor vests as options, and do not have the durathread vest
- Scientist is missing the Roboticist lab coat (The role has a roboticist jumpsuit option)
- Bartender is missing their beer goggles
- Musician has sunglasses by default in roles/musician.yml, so none of the universal glasses options work

Haltell committed:
Fixed Commenting

Properly Commented Out MedicalGloves/Shoes instead of deleting them.

WarMechanic commented:
> Sec jobs have the two deprecated armor vests as options, and do not have the durathread vest

I don't think this is that bad, from my understanding people like the additional cosmetic variety that they provide.

WarMechanic commented:
A change that I've been wanting to make is to make less experienced job uniforms available to more experience / command jobs:
For example, both station engineer and atmos tech gain access to yellow jumpskirts. Chief engineer would get access to station engineer, atmos tech and yellow jumpskirt loadout options.
I could do this in a separate PR. but I'm happy for you to do it here.

Haltell commented:
> A change that I've been wanting to make is to make less experienced job uniforms available to more experience / command jobs: For example, both station engineer and atmos tech gain access to yellow jumpskirts. Chief engineer would get access to station engineer, atmos tech and yellow jumpskirt loadout options. I could do this in a separate PR. but I'm happy for you to do it here.

I'd like to see it too! But it more or less depends on if the people approving this want those options in game. It's otherwise a fairly simple addition of just adding more lines, but we've had issues in the past of heads trying to hide themselves and practically abandon the station during nukies, or similar.

Haltell committed:
More Security Options

Added Blue Jumpsuit to Sec Officer
Added Red Beret to Sec Officer and Cadet
Added Durathread Vest to Officers
Added Red Scarf to all of Security

Timemaster99 commented:
> > Sec jobs have the two deprecated armor vests as options, and do not have the durathread vest
> 
> I don't think this is that bad, from my understanding people like the additional cosmetic variety that they provide.

They were replaced by the durathread vest long ago for balancing, given their different armor values. They were consequently removed from maps and lockers. Discussion would be required to keep them, since they are currently unobtainable anywhere besides loadouts

Haltell committed:
Logistics Officer Beret

Fixed LogiOfficerBeret not being in loadout_groups.yml

DangerRevolution commented:
> > > Sec jobs have the two deprecated armor vests as options, and do not have the durathread vest
> > 
> > 
> > I don't think this is that bad, from my understanding people like the additional cosmetic variety that they provide.
> 
> They were replaced by the durathread vest long ago for balancing, given their different armor values. They were consequently removed from maps and lockers. Discussion would be required to keep them, since they are currently unobtainable anywhere besides loadouts

The new vests are resprites because Wizden use the ugliest sprites imaginable, and I imagine no one has bothered updating the armour stats on them. It was never removed as a change exclusively to armour values.

Timemaster99 commented:
> > > > Sec jobs have the two deprecated armor vests as options, and do not have the durathread vest
> > > 
> > > 
> > > I don't think this is that bad, from my understanding people like the additional cosmetic variety that they provide.
> > 
> > 
> > They were replaced by the durathread vest long ago for balancing, given their different armor values. They were consequently removed from maps and lockers. Discussion would be required to keep them, since they are currently unobtainable anywhere besides loadouts
> 
> The new vests are resprites because Wizden use the ugliest sprites imaginable, and I imagine no one has bothered updating the armour stats on them. It was never removed as a change exclusively to armour values.

They would have changed the sprites instead of making new prototypes, I would strongly presume. They also do not have the same armor values. Here's some images
![image](https://github.com/DeltaV-Station/Delta-v/assets/57200767/4252a942-5066-4401-ac18-c839d6370491)
![image](https://github.com/DeltaV-Station/Delta-v/assets/57200767/217c8a2d-fa35-4ea3-958b-bb232c5368ab)
![image](https://github.com/DeltaV-Station/Delta-v/assets/57200767/fc5c8e20-4764-43d5-843a-0f280b271641)
![image](https://github.com/DeltaV-Station/Delta-v/assets/57200767/7d2db9ab-b3d5-4da0-92c7-dcff4ea7fd14)

Haltell commented:
> > > > Sec jobs have the two deprecated armor vests as options, and do not have the durathread vest
> > > 
> > > 
> > > I don't think this is that bad, from my understanding people like the additional cosmetic variety that they provide.
> > 
> > 
> > They were replaced by the durathread vest long ago for balancing, given their different armor values. They were consequently removed from maps and lockers. Discussion would be required to keep them, since they are currently unobtainable anywhere besides loadouts
> 
> The new vests are resprites because Wizden use the ugliest sprites imaginable, and I imagine no one has bothered updating the armour stats on them. It was never removed as a change exclusively to armour values.

Do you want me to comment them out quickly, as well?

Haltell committed:
Command Loadout Options

Added Glove Options for Captain and HoP
Added More Uniform Options for HoP
Added More Outer Clothing Options for HoP
Added Shoe Options for Captain and HoP

Haltell committed:
Winter Options

Scarfs added to most roles, with the following exceptions:
- Assistant Roles
- Captain
- Chef
- Mantis
- Prison Guard
- Station Specific Jobs (except Corpsman)

Stethoscope added as option for Medical Doctors, CMOs, and Paramedics

Salvager Cape added and Miner Winter Boots replaced Logi Winter Boots as option for salvagers

Heads of Staff Winter Boots Added for Respective Heads

Paramedic Winter Boots added for Paramedics

Removed Requirement for Engineers to wear a hat
Added hats option for Atmospheric Technicians

Added Roboticist Coat Option for Scientists

Haltell committed:
Musician Eyewear

Fixed musician's customizable eyewear by removing their default sunglasses.

Musician no longer able to buy expensive sunglasses.

Cheap sunglasses available through glasses loadout group with 2 hours of musician time.

Haltell committed:
Beer Goggles

Changed Bartender's Glasses Options to include beer goggles.

Haltell committed:
Update glasses.yml

Add blankspace to the end of glasses.yml

Fix the ad hoc cheap sunglasses timer

Haltell committed:
Update glasses.yml

Fix sunglasses timer

Haltell committed:
Merge branch 'master' into loadout-options

Timemaster99 commented:
Chemist also requires their formal jumpsuit and tie

Haltell committed:
Clothing Options

Added extra clothing options for Chemist and Musician. Formal attire and red dress respectively.

Axionyxx committed:
Merge branch 'master' into loadout-options

Axionyxx reviewed (approved):
Looks good to me, but I'd prefer someone else to read over them too

therealDLondon commented:
*rubber stamp sound effect*
Headmin Approv!
